package.json

```diff
@@ -2,54 +2,74 @@
   "name": "stylelint-config-prettier-scss",
   "version": "0.0.1",
   "description": "Turns off all SCSS rules that are unnecessary or might conflict with Prettier.",
-  "scripts": {
```

Comment:
Most changes to this file are just re-ordering the fields, not changing them. Will comment what actually changed.